swap_meet/vendor.py

```diff
@@ -1,2 +1,104 @@
+
+####################### WAVE 1 #############################
+
 class Vendor:
-    pass
\ No newline at end of file
+    # inventory is an empty list
+    def __init__(self, inventory=None):
+
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
```

Comment:
Great work setting out default parameter to `None` and resetting it to an empty list if it's not passed in!